test/integration/crcsuite/crcsuite.go

```diff
@@ -114,6 +114,33 @@ func FeatureContext(s *godog.Suite) {
 			fmt.Println(err)
 			os.Exit(1)
 		}
+
+		if !bundleEmbedded {
+			if _, err := os.Stat(bundleLocation); err != nil {
+				if !os.IsNotExist(err) {
+					fmt.Printf("Unexpected error obtaining the bundle %v.\n", bundleLocation)
+					os.Exit(1)
+				}
+				// Obtain the bundle to current dir
+				fmt.Println("Obtaining bundle...")
+				bundleLocation, err = DownloadBundle(bundleLocation, ".")
+				if err != nil {
+					fmt.Printf("Failed to obtain CRC bundle, %v\n", err)
+					os.Exit(1)
+				}
+				fmt.Println("Using bundle:", bundleLocation)
+			} else {
+				fmt.Println("Using existing bundle:", bundleLocation)
+			}
+		}
+	})
+
+	s.BeforeScenario(func(pickle *messages.Pickle) {
+		// copy data/config files to test dir
+		err := CopyFilesToTestDir()
```

Comment:
Should this be in `BeforeSuite`? It copies [this folder's](https://github.com/code-ready/crc/tree/master/test/testdata) contents to the execution folder (i.e. `crc/test/integration/out/run`). The execution folder should stay put throughout all Features - the previous change with bundle download relies on this iirc?

Reply:
`BeforeSuite` is deprecated and we should limit using deprecated features.

Reply:
I thought `BeforeFeature` was deprecated, thread [here](https://www.gitmemory.com/issue/cucumber/godog/335/665641230).

Reply:
Ah right so `BeforeFeature` is deprecated for bundle I changed the logic which can't be done for these testdata files.